babel-preset/configs/hot.js

```diff
@@ -0,0 +1,28 @@
+/**
```

Comment:
nit: Maybe this should be named HMR as hot loading is a usecase of hot module replacement. Don't feel strong about this so feel free to leave it as is